tests/testthat/_snaps/4.2/dq_and_pdde/test-dq_and_pdde-003.json

```diff
@@ -1976,347 +1977,347 @@
-        "2021-11-04",
-        "2021-11-04",
-        "2021-11-08",
-        "2021-11-09",
-        "2021-11-11",
-        "2021-11-12",
-        "2021-11-21",
-        "2021-11-22",
-        "2021-11-24",
-        "2021-12-01",
-        "2021-12-03",
-        "2021-12-07",
-        "2021-12-14",
-        "2021-12-15",
-        "2021-12-22",
-        "2021-12-22",
-        "2021-12-27",
-        "2021-12-28",
-        "2021-12-30",
-        "2022-01-01",
-        "2022-01-10",
-        "2022-01-12",
-        "2022-01-19",
-        "2022-01-24",
-        "2022-01-24",
-        "2022-01-24",
-        "2022-01-25",
-        "2022-01-25",
-        "2022-01-27",
-        "2022-01-27",
-        "2022-01-28",
-        "2022-01-31",
-        "2022-02-01",
-        "2022-02-02",
-        "2022-02-02",
-        "2022-02-03",
-        "2022-02-03",
-        "2022-02-05",
-        "2022-02-07",
-        "2022-02-07",
-        "2022-02-07",
-        "2022-02-10",
-        "2022-02-10",
-        "2022-02-10",
-        "2022-02-19",
-        "2022-02-26",
-        "2022-03-01",
-        "2022-03-01",
-        "2022-03-01",
-        "2022-03-03",
-        "2022-03-03",
-        "2022-03-07",
-        "2022-03-11",
-        "2022-03-13",
-        "2022-03-15",
-        "2022-03-21",
-        "2022-03-24",
-        "2022-03-24",
-        "2022-03-25",
-        "2022-04-01",
-        "2022-04-03",
-        "2022-04-10",
-        "2022-04-18",
-        "2022-04-18",
-        "2022-04-19",
-        "2022-04-21",
-        "2022-04-22",
-        "2022-04-26",
-        "2022-04-26",
-        "2022-04-30",
-        "2022-05-02",
-        "2022-05-09",
-        "2022-05-10",
-        "2022-05-13",
-        "2022-05-16",
-        "2022-05-16",
-        "2022-05-17",
-        "2022-05-18",
-        "2022-05-24",
-        "2022-05-25",
-        "2022-05-26",
-        "2022-05-26",
-        "2022-05-28",
-        "2022-06-01",
-        "2022-06-01",
-        "2022-06-04",
-        "2022-06-08",
-        "2022-06-10",
-        "2022-06-10",
-        "2022-06-10",
-        "2022-06-10",
-        "2022-06-10",
-        "2022-06-13",
-        "2022-06-13",
-        "2022-06-25",
-        "2022-06-27",
-        "2022-06-29",
-        "2022-06-30",
-        "2022-07-01",
-        "2022-07-05",
-        "2022-07-12",
-        "2022-07-14",
-        "2022-07-15",
-        "2022-07-19",
-        "2022-07-20",
-        "2022-08-04",
-        "2022-08-04",
-        "2022-08-04",
-        "2022-08-16",
-        "2022-08-16",
-        "2022-08-18",
-        "2022-08-20",
-        "2022-08-21",
-        "2022-08-23",
-        "2022-08-23",
-        "2022-08-24",
-        "2022-08-26",
-        "2022-08-28",
-        "2022-08-28",
-        "2022-09-01",
-        "2022-09-08",
-        "2022-09-10",
-        "2022-09-16",
-        "2022-09-20",
-        "2022-09-21",
-        "2022-09-21",
-        "2022-09-21"
+        "2019-06-03T00:00:00Z",
+        "2019-07-15T00:00:00Z",
```

Comment:
this should not have times. we fixed this in dev. need to pull relevant code into branch so snapshot doesn't have times.